app/src/main/java/com/grarak/kerneladiutor/fragments/kernel/SoundFragment.java

```diff
@@ -294,4 +301,46 @@ public void onMove(SeekBarView seekBarView, int position, String value) {
         items.add(volumeGain);
     }
 
+   private void headphoneFlarInit(List<RecyclerViewItem> items) {
+
+        TitleView title = new TitleView();
+        title.setText(getString(R.string.sound_control));
+
+        SeekBarView headphoneFlar = new SeekBarView();
+        headphoneFlar.setTitle(getString(R.string.headphone_gain));
+        headphoneFlar.setItems(Sound.getHeadphoneFlarLimits());
+        headphoneFlar.setProgress(Sound.getHeadphoneFlarLimits().indexOf(Sound.getHeadphoneFlar()));
+        headphoneFlar.setOnSeekBarListener(new SeekBarView.OnSeekBarListener() {
+            @Override
+            public void onStop(SeekBarView seekBarView, int position, String value) {
+                Sound.setHeadphoneFlar(value, getActivity());
+            }
+
+            @Override
+            public void onMove(SeekBarView seekBarView, int position, String value) {
+            }
+        });
+        items.add(title);
```

Comment:
Nah nah its ok for me, i picked your commit and removed the titleview because for my eyes looks unecessary.